scripts/fuzz_opt.py

```diff
@@ -117,6 +117,17 @@ def randomize_feature_opts():
     print('randomized feature opts:', ' '.join(FEATURE_OPTS))
 
 
+def update_feature_opts(wasm):
+    global FEATURE_OPTS
+    # we will re-compute the features; leave all other things as they are
+    EXTRA = [x for x in FEATURE_OPTS if not x.startswith('--enable') and not x.startswith('--disable') and x != '--all-features']
```

Comment:
What is `EXTRA` for? Why is `--all-features` excluded but not `-all` or `--mvp-features` or `-mvp`?

Reply:
Ah, I just handled the actually used ones. But good point, I generalized it now.